hist/hist/src/TFormula.cxx

```diff
@@ -998,50 +1000,113 @@ void TFormula::FillVecFunctionsShurtCuts() {
 ///    eg.
 ///    varpol2(5) will be replaced with: [5] + [6]*var + [7]*var^2
 ///    Empty name is treated like variable x.
+///    Extended format also supports direct variable specification: pol2(x, 0) or pol(x, [A])
+///    Traditional Syntax: polN - Polynomial ofreturn it->second;(x)
+///    ypol1             [p0]+[p1]*y
+///    pol1(x, 0)        [p0]+[p1]*x
+///    pol2(y, 1)        [p1]+[p2]*y+[p3]*TMath::Sq(y)
```

Comment:
Let's indent this, so doxygen will display this like code (see [here](https://www.doxygen.nl/manual/markdown.html#md_codeblock)):
```suggestion
///          ypol1             [p0]+[p1]*y
///          pol1(x, 0)        [p0]+[p1]*x
///          pol2(y, 1)        [p1]+[p2]*y+[p3]*TMath::Sq(y)
```